Makefile

```diff
@@ -261,10 +261,12 @@ clean: clean-certs clean-containers clean-slice ## Stops all containers and remo
 	$(RM) *.box
 
 test: ## executes the test script on a master
+	# This command is broken, as per
```

Comment:
These comments are confusing. Doesn't this change fix the issue? If so, why do we need to comment about the jira?